web_client/src/components/WindowWidget.vue

```diff
@@ -66,8 +66,16 @@ export default defineComponent({
 
     function autoRange() {
       if (windowLocked.value) return;
-      currentRange.value = [widthMin.value, widthMax.value];
-      updateFromRange(currentRange.value);
+      const data = props.representation.getInputDataSet();
+      const distribution = data.computeHistogram(data.getBounds());
+      console.log(distribution);
```

Comment:
```suggestion
```

Reply:
Removed in [a840488](https://github.com/OpenImaging/miqa/pull/567/commits/a840488c70d450e1527858317c85212cc7333467)